addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -103,4 +103,88 @@
 
             </t>
         </template>
+        <template id="auth_signup.reset_password_email" name="User Reset Password">
```

Comment:
:socks: maybe to move in a specific file to separate qweb used for frontend and qweb used for emails ? like 'auth_signup_templates_email' ?